tests/BrowserTests.php

```diff
@@ -221,6 +221,17 @@ public function content_assertions(): void
         ;
     }
 
+    /**
+     * @test
+     */
+    public function assert_see_in_multiple_elements(): void
+    {
+        $this->browser()
+            ->visit('/page1')
+            ->assertNotSeeIn('ul li', 'list 2') // todo I'd like this to work in 2.x
```

Comment:
Aren't tests passing? 😅

Reply:
They are but I want this to fail in 2.x. It is confusing haha

Maybe I'll make it fail and just keep this PR open until 2.x

Reply:
Make it risky or something like it?